### PR TITLE
Standardize Docs: web-frame, web-view-tag, window-open

### PR DESCRIPTION
**Pull request by jlord**
Next batch of docs, re: [styleguide](https://github.com/atom/electron/blob/master/docs/styleguide.md).

These are pretty basic edits. Ready for :eyes:

### CODE REVIEW

File: docs/api/web-frame.md [hunk 1]
```diff
@@ -1,45 +1,50 @@
...
 
 Returns the current zoom factor.
 
-## webFrame.setZoomLevel(level)
+### `webFrame.setZoomLevel(level)`
 
 * `level` Number - Zoom level
 
-Changes the zoom level to the specified level, 0 is "original size", and each
+Changes the zoom level to the specified level. The "original size" is 0 and each
```
Comment (kevinsawicki) [inline docs/api/web-frame.md]: I'm not sure `"original size"` needs to be in quotes here.

File: docs/api/web-view-tag.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
-# `<webview>` tag
+# The `<webview>` tag
 
 Use the `webview` tag to embed 'guest' content (such as web pages) in your
-Electron app. The guest content is contained within the `webview` container;
-an embedder page within your app controls how the guest content is laid out and
+Electron app. The guest content is contained within the `webview` container.
+An embedder page within your app controls how the guest content is laid out and
```
Comment (kevinsawicki) [inline docs/api/web-view-tag.md]: Would `embedded` instead of `embedder` be better?

File: docs/api/web-view-tag.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 rendered.
 
-Different from the `iframe`, the `webview` runs in a separate process than your
-app; it doesn't have the same permissions as your web page and all interactions
+Different from an `iframe`, the `webview` runs in a separate process than your
```
Comment (kevinsawicki) [inline docs/api/web-view-tag.md]: Maybe `Unlike an`, instead of `Different from an`?

File: docs/api/web-view-tag.md [hunk 4]
```diff
@@ -69,11 +71,11 @@ The `src` attribute can also accept data URLs, such as
 
 If "on", the `webview` container will automatically resize within the
 bounds specified by the attributes `minwidth`, `minheight`, `maxwidth`, and
-`maxheight`. These contraints do not impact the `webview` UNLESS `autosize` is
+`maxheight`. These constraints do not impact the `webview` UNLESS `autosize` is
```
Comment (kevinsawicki) [inline docs/api/web-view-tag.md]: Can we normal case `UNLESS`? :scream:

Comment (kevinsawicki) [inline docs/api/web-view-tag.md]: Or use bold/italic

File: docs/api/web-view-tag.md [hunk 5]
```diff
@@ -102,9 +104,9 @@ will be loaded by `require` in guest page under the hood.
 
 When the guest page doesn't have node integration this script will still have
 access to all Node APIs, but global objects injected by Node will be deleted
-after this script has done execution.
+after this script has completed execution.
```
Comment (kevinsawicki) [inline docs/api/web-view-tag.md]: Maybe `finished executing` instead of `completed execution`?

File: docs/api/web-view-tag.md [hunk 6]
```diff
@@ -138,194 +144,195 @@ webview.addEventListener("dom-ready", function() {
...
 
-Returns whether guest page is still loading resources.
+Returns a boolean whether guest page is still loading resources.
 
-### `<webview>`.isWaitingForResponse()
+### `<webview>.isWaitingForResponse()`
 
-Returns whether guest page is waiting for a first-response for the main resource
-of the page.
+Returns a boolean whether guest page is waiting for a first-response for the
```
Comment (kevinsawicki) [inline docs/api/web-view-tag.md]: Should it be `the guest page` here and below?

File: docs/api/web-view-tag.md [hunk 7]
```diff
@@ -138,194 +144,195 @@ webview.addEventListener("dom-ready", function() {
...
+### `<webview>.executeJavaScript(code, userGesture)`
 
 * `code` String
-* `userGesture` Boolean - Default false
+* `userGesture` Boolean - Default `false`.
 
-Evaluates `code` in page. If `userGesture` is set will create user gesture context,
-HTML api like `requestFullScreen` which require user action can take advantage
-of this option for automation.
+Evaluates `code` in page. If `userGesture` is set, it will create user gesture
```
Comment (kevinsawicki) [inline docs/api/web-view-tag.md]: `in the page`?

**Comment by jlord** [comment]
Thanks @kevinsawicki! Made the changes :+1:

**Pull request merged**